### ISSUE TITLE
Ulauncher doest not running

### ISSUE THREAD

**Issue by ibnumusyaffa** [Ulauncher/Ulauncher#443]
```
(ulauncher:18499): dbind-WARNING **: 22:35:56.801: Couldn't register with accessibility bus: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
/usr/lib/python3/dist-packages/ulauncher/main.py:8: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk
2019-10-02 22:35:57,129 | ERROR | ulauncher: except_hook() | Uncaught exception
Traceback (most recent call last):
  File "/usr/bin/ulauncher", line 28, in <module>
    main()
  File "/usr/lib/python3/dist-packages/ulauncher/main.py", line 125, in main
    window = UlauncherWindow.get_instance()
  File "/usr/lib/python3/dist-packages/ulauncher/utils/decorator/singleton.py", line 19, in wrapper
    instance = fn(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/ulauncher/ui/windows/UlauncherWindow.py", line 54, in get_instance
    return cls()
  File "/usr/lib/python3/dist-packages/ulauncher/ui/windows/UlauncherWindow.py", line 64, in __new__
    new_object.finish_initializing(builder)
  File "/usr/lib/python3/dist-packages/ulauncher/ui/windows/UlauncherWindow.py", line 99, in finish_initializing
    self.show_results([])
  File "/usr/lib/python3/dist-packages/ulauncher/ui/windows/UlauncherWindow.py", line 323, in show_results
    result_items = AppStatDb.get_instance().get_most_frequent(3)
  File "/usr/lib/python3/dist-packages/ulauncher/utils/decorator/singleton.py", line 19, in wrapper
    instance = fn(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/ulauncher/search/apps/AppStatDb.py", line 18, in get_instance
    return cls(os.path.join(DATA_DIR, 'app_stat_v2.db')).open()
  File "/usr/lib/python3/dist-packages/ulauncher/utils/db/KeyValueDb.py", line 35, in open
    self.set_records(pickle.load(_in))
EOFError: Ran out of input
```

OS : Linux mint 19

**Comment by knsrky** [comment]
same issue...

**Comment by agravelot** [comment]
Facing same kind of issue on Fedora 30 and 31.

```
ulauncher                                                                    14.8s  mer. 09 oct. 2019 22:14:31 CEST
/usr/lib/python3.7/site-packages/ulauncher/main.py:8: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '4.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk
Traceback (most recent call last):
  File "/usr/bin/ulauncher", line 27, in <module>
    from ulauncher.main import main
  File "/usr/lib/python3.7/site-packages/ulauncher/main.py", line 13, in <module>
    gi.require_version('Gtk', '3.0')
  File "/usr/lib64/python3.7/site-packages/gi/__init__.py", line 121, in require_version
    (namespace, loaded_version))
ValueError: Namespace Gtk is already loaded with version 4.0
```

```
ulauncher-toggle                            
Error org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
```

**Comment by gornostal** [comment]
The issue should have been fixed with Ulauncher v5.6.0

Please reopen if needed
